examples/data-sources/ovirt_disk_attachments/data.tf

```diff
@@ -0,0 +1,10 @@
+data "ovirt_disk_attachments" "set" {
```

Comment:
Please run `go generate` to generate the docs and make sure the docs make sense with this example.